My diplom_1: tests, pom.xml, report jacoco.

AlenaTerletskaya opened the pull request:
В pom.xml подключены библиотеки:  mockito-core, junit, JUnitParams.
 В pom.xml добавлен плагин: jacoco-maven-plugin.
 Написаны тесты на классы Bun, Burger, Ingredient, IngredientType: 
 BunTest, BurgerTest, IngredientParamTest, IngredientTest, IngredientTypeTest.
 Добавлен отчет jacoco.

AlenaTerletskaya committed:
В pom.xml подключены библиотеки: mockito-core, junit, JUnitParams;
 В pom.xml добавлен плагин: jacoco-maven-plugin.
 Написаны тесты на классы Bun, Burger, Ingredient, IngredientType:
 BunTest, BurgerTest, IngredientParamTest, IngredientTest, IngredientTypeTest.
 Добавлен отчет jacoco.

nikolay-977 commented on src/test/java/IngredientParamTest.java:
```diff
@@ -0,0 +1,48 @@
+    }
+
+    // Метод для получения данных
+    @Parameterized.Parameters
```
Можно улучшить: В параметризованных тестах для аннотации @Parameterized.Parameters лучше использовать аргумент name: @Parameterized.Parameters(name = "Тестовые данные: {0} {1}"), где {0}, {1} - индексы параметров. Это повысит информативность проверки.

nikolay-977 commented on src/test/java/IngredientTest.java:
```diff
@@ -0,0 +1,44 @@
+        mockIngredient.getType();
+        Mockito.verify(mockIngredient).getType();
+    }
+
```
Необходимо исправить: убери лишние пропуски строк. Между фигурными скобками и после аннотаций пропуски не нужны. Пропуски строк должны быть однообразными.

AlenaTerletskaya committed:
В тестовых классах убрала лишние пропуски строк, сделала их однообразными.
В классах BurgerTest и IngredientTest убрала не используемую аннотацию @RunWith(JUnitParamsRunner.class).
В классе IngredientParamTest добавила к аннотации @Parameterized.Parameters аргумент name.
